src/frontend/src/catalog/schema_catalog.rs

```diff
@@ -776,8 +782,22 @@ impl SchemaCatalog {
         self.source_by_id.get(source_id)
     }
 
-    pub fn get_sink_by_name(&self, sink_name: &str) -> Option<&Arc<SinkCatalog>> {
-        self.sink_by_name.get(sink_name)
+    pub fn get_sink_by_name(
+        &self,
+        sink_name: &str,
+        bind_creating: bool,
+    ) -> Option<&Arc<SinkCatalog>> {
+        self.sink_by_name
+            .get(sink_name)
+            .filter(|s| bind_creating || s.is_created())
+    }
+
+    pub fn get_any_sink_by_name(&self, sink_name: &str) -> Option<&Arc<SinkCatalog>> {
```

Comment:
The method name `get_any_sink` indicates that it includes both creating and created sinks, but it actually returns only created sinks. Is this expected?